.meta.toml

```diff
@@ -0,0 +1,176 @@
+# Generated from:
+# https://github.com/plone/meta/tree/master/config/default
+[meta]
+template = "default"
+commit-id = "50c0e759"
+
+[editorconfig]
+extra_lines = """
+[*.bat]
+indent_style = tab
+end_of_line = crlf
+
+[LICENSE]
+insert_final_newline = false"""
+
+[setup]
+check_manifest_ignore = """
+    *.cfg
+    constraints_plone52.txt
+    constraints_plone60.txt
+    constraints.txt
+    fix-converted-myst.py
+    Makefile
+    netlify.toml
+    requirements-docs.txt
+    requirements.txt"""
+extra_lines = """
+[coverage:run]
+branch = True
+source =
+    src
+
+omit =
+    *.rst
+    */doctests/*
+    */tests/*
+
+[coverage:report]
+precision = 2
+include = src
+omit =
+    *.rst
+    */doctests/*
+    */tests/*
+
+[coverage:html]
+directory = _build/coverage"""
+
+[tox]
+envlist_lines = """
+    py{38,39}-plone{52}
```

Comment:
We should not test Plone 5.2 on Python 3.9.  But that is already happening on current master I think, so could be handled separately.